modules/AOR_Reports/tpls/EditViewFooter.tpl

```diff
@@ -268,6 +268,21 @@
             updateChartDimensionSelects();
 
         });
+        $('#addChartButton').css({
```

Comment:
Would it not make more sense use the button class instead of duplicating the jQuery selector and css? you can accomplish the same thing here by adding class="button" line 373.